zproject/settings.py

```diff
@@ -1030,6 +1038,16 @@ def get_secret(key):
             'handlers': ['file'],
             'propagate': False,
         },
+        'django.server': {
+            'propagate': False,
+            'filters': ['skip_200_and_304'],
+        },
+        'django.template': {
+            'handlers': ['console'],
+            'filters': ['require_debug_true'],
+            'level': 'DEBUG',
+            'propgate': False,
+        },
```

Comment:
Hmm, my read of https://docs.djangoproject.com/en/1.10/topics/logging/#examples is that having `propogate: False` here suggests that _anything_ that matches this handler will get eaten by it, even if they have a higher log level than `DEBUG`.  But it's a bit ambigous; maybe worth a quick test?